assets/app/features/upgrade/i18n/en.json

```diff
@@ -119,6 +119,19 @@
         "form": {
           "check-again": "Check Again"
         }
+      },
+      "database-configuration": {
+        "name": "",
+        "title": "Create or Connect to an OpenStack PostgreSQL Database",
+        "form": {
+          "username": "Username",
+          "password": "Password",
+          "create-database": "Create New Database",
+          "server": "server",
+          "port": "port",
+          "table-prefix": "Table_Prefix",
```

Comment:
Please keep using capital first letters and space, not underscores for consistency across all the labels.